madushag/MyFarm
# Feature/myf 55 view form template for signup

## This pull request contributes to MYF-57, MYF-55, and MYF-17:

### What is included in this pull request
- adds basic registration page
- adds basic form functionality to allow a user to create a new account (username, password, email)

### How to test
- Launch the application, either with docker or on local machine
- Go to localhost:8000/register in your browser
- You should see a basic registration screen
- Fill out the info and you should be redirected to the homepage.

### Remaining work
- I still need to stub out a new page to redirect users
- additional form validation needed
- incorporate groups as part of signup process
- At the moment users can only register, I need to repurpose view to also allow sign-in functionality if the user has already registered.
